doc/tutorials/13-charged_rod/13-charged_rod.ipynb

```diff
@@ -0,0 +1,270 @@
+{
```

Comment:
<p>i'm not sure if we want to keep to the PEP8 convention, but if so, all those variables need to be CAPITALIZED</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/espressomd/espresso/pull/3886/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>